web/eslint.config.mjs

```diff
@@ -0,0 +1,35 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReactConfig from "eslint-plugin-react/configs/recommended.js";
+
+export default [
+    { files: ["**/*.{js,mjs,cjs,ts,jsx,tsx}"] },
```

Comment:
Weirdly enough this does not stop `.snap` from being touched? I don't think we want to lint auto-generated snapshots, or do we?

Reply:
Huh - snap files are not linted for me?

Reply:
Oh, never mind then. I though the changes in the snap files in this PR were caused by eslint. But I assume some other changed caused that?

Reply:
Yes, those are test refactors / upgrades.